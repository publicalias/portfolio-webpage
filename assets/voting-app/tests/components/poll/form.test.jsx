"use strict";

//local imports

const Form = require("../../../scripts/components/poll/form");

//global imports

const { reactTests } = require("test-helpers/react-tests");

//node modules

const React = require("react");

const { shallow } = require("enzyme");

//setup

beforeAll(reactTests.setup);

//form

test("form should match snapshot", () => {

  const wrapper = shallow(<Form />);

  expect(wrapper).toMatchSnapshot();

});
