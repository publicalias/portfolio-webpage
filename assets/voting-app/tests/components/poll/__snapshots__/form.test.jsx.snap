// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`form should match snapshot 1`] = `
<div
  className="c-ui__form"
/>
`;
